Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -3734,7 +3735,7 @@ module.exports = {
         _('Collision'),
         _('Test if two objects collide.'),
         _('_PARAM0_ is colliding with _PARAM2_'),
-        '',
+        _('Collisions'),
         'res/physics32.png',
```

Comment:
The advantage to leave no category is that it is on top of the list. It makes sens as it might be the most used condition.
The group "Collision" (without an "s") could be used to merge it with the built-in conditions.